.ci/no_old_refs.sh

```diff
@@ -2,27 +2,28 @@
 # Attention, there is no "-x" to avoid problems on Travis
 set -e
 
-MENTIONED_IDS=/tmp/mentioned_ids
-CLOSED_ISSUES=/tmp/failed
-API_ISSUE_LINK_PREFIX="https://api.github.com/repos/checkstyle/checkstyle/issues"
-ISSUE_LINK_PREFIX="https://github.com/checkstyle/checkstyle/issues"
+MENTIONED_ISSUES=/tmp/mentioned_issues
+CLOSED_ISSUES=/tmp/failed_issues
+API_GITHUB_PREFIX="https://api.github.com/repos"
+GITHUB_HOST="https://github.com"
 
 # collect issues where full link is used
-grep -Pohr "(after|[Tt]il[l]?) $ISSUE_LINK_PREFIX/\d{1,5}" . \
-  | sed -e 's/.*issues\///' >> $MENTIONED_IDS
+grep -Pohr "(after|[Tt]il[l]?) $GITHUB_HOST/[a-z]+/[a-z]+/issues/\d{1,5}" . \
```

Comment:
The pattern `[a-z]` is not enough here. There may be dashes and numbers in the uri, for example:
```
https://github.com/junit-team/junit5/issues/1786
https://github.com/Azure/actions/issues/2
https://github.com/Abhishek-kumar09/susi_server/1
https://github.com/strkkk/strkkk.github.io/issues/1
```

Reply:
Good catch, fixed.
By the way, all mentioned repos in code have only lowercase :)